arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -106,7 +107,7 @@ public async Task<GetGraphResponse> GetGraphAsync(string graphName)
         /// <returns></returns>
         public async Task<GetVertexCollectionsResponse> GetVertexCollections(string graph)
```

Comment:
Minor consistency thing but how about calling this `graphName` like the other methods?